pkg/storage/mysql/mysqlstore.go

```diff
@@ -149,10 +149,10 @@ func (s *sqlDBStore) Put(k string, v []byte) error {
 		return storage.ErrKeyRequired
 	}
 
-	//nolint: gosec create upsert query to insert the record, checking whether the key is already mapped to a value in the store.
-	createStmt := "INSERT INTO " + s.tableName + " VALUES (?, ?) ON DUPLICATE KEY UPDATE value=?"
+	// create upsert query to insert the record, checking whether the key is already mapped to a value in the store.
+	createStmt := "INSERT INTO ? VALUES (?, ?) ON DUPLICATE KEY UPDATE value=?"
```

Comment:
@talwinder50

Reply:
https://github.com/go-sql-driver/mysql/issues/848

I have already tried This query wont work see the spec they dont support placeholders on table name they only support on where and like. If we run this against the real mysql it will throw syntax error

Reply:
@talwinder50 thanks. Looks like we need to rollback this PR....